framework/include/fwk_module.h

```diff
@@ -222,6 +222,27 @@ struct fwk_module {
      */
     int (*start)(fwk_id_t id);
 
+    /*!
+     * \brief Pointer to the stop function.
+     *
+     * \details This function is called by the framework for the module and then
+     *      for all of its elements during the stop stage. Elements are
+     *      stopped in the order they are declared in the module configuration
+     *      data.
+     *
+     *      The framework does not mandate a particular purpose for this
+     *      function. It may be used to perform any final processing of the
+     *      module and its elements before entering the suspend phase.
+     *
+     * \note This function is \b optional.
```

Comment:
Please add a note that this is not to stop/start at runtime but rather for terminating the application.

Reply:
ok will add:
This function is \b optional and aims to release software resources or to set hardware resources in a proper state before terminating the SCP software